README.md

```diff
@@ -720,9 +720,11 @@ public class Variant
 }
 ```
 
-### Getting a Feature's Variant
+### Getting Variants

```

Comment:
I think it would be better if the code snippet came directly after this part "For each feature, a variant can be retrieved using the IVariantFeatureManager's GetVariantAsync method."

then after the code snippet the extra information was provided.

Reply:
I was thinking in our doc it might be helpful to mention that .NET makes it easy to bind IconfigurationSection to a type. After all, this is how I expect most people to consume a features variants. After seeing this, I think a second code snippet could be provided. something like below

---

Once a variant is retrieved, the configuration of a variant can be used directly as an `IConfigurationSection` from the variants `Configuration` property. Another option is to bind the configuration to a type using .NET's configuration binding pattern.

```
IConfigurationSection variantConfiguration = variant.Configuration;

MyFeatureSettings settings = variant
    .Configuration
    .Bind<MyFeatureSettings>();
```

Reply:
``` C#
MyFeatureSettings settings = variantConfiguration.Get<MyFeatureSettings>() ?? new MyFeatureSettings();
```
Or
```
MyFeatureSettings settings = new MyFeatureSettings();
variantConfiguration.Bind(settings);
```